packages/main/src/List.ts

```diff
@@ -1349,6 +1388,24 @@ class List extends UI5Element {
 		}
 	}
 
+	async focusOnFirstLoadedItem() {
+		const currentLength = this.getItems().length;
+
+		this.loadMore();
+
+		await renderFinished();
```

Comment:
While this works, I wonder if we fetch asynchronously real data/items, if it's going to work if the items arrive after a few seconds. May be we should think a more robust way of determine whether or not new items have arrived and when.